includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -798,6 +798,29 @@ public function detach_payment_method( $payment_method_id ) {
 		);
 	}
 
+	/**
+	 * Records a new Terms of Service agreement.
+	 *
+	 * @param string $source     A string, which describes where the merchant agreed to the terms.
+	 * @param string $user_name  The user_login of the current user.
+	 * @param string $user_ip    IP address of the current user.
+	 * @param string $user_agent User agent string for the current user.
+	 *
```

Comment:
🔍 `@return` doc statement is missing now.

Reply:
Added here: https://github.com/Automattic/woocommerce-payments/pull/993/commits/e96212bdf67f2f988ab11d725768c04da1156dd6, thanks for noticing!